operating/manage-vulnerabilities/vulnerability-management.adoc

```diff
@@ -7,60 +7,46 @@ include::modules/common-attributes.adoc[]
 
 include::modules/vulnerability-management-process.adoc[leveloffset=+1]
-
 include::modules/vulnerability-management-asset-assessment.adoc[leveloffset=+2]
+include::modules/vulnerability-management-prioritizing.adoc[leveloffset=+2]
+include::modules/vulnerability-management-assess-exposure.adoc[leveloffset=+2]
+include::modules/vulnerability-management-take-action.adoc[leveloffset=+2]
 
 [id="viewing-vulnerabilities"]
 == Viewing vulnerabilities
 
-{product-title-short} provides the following methods to view vulnerabilities discovered in your system:
+Historically, {product-title-short} provided a view of vulnerabilities discovered in your system in the *Vulnerability Management Dashboard*. With the dashboard, you can view vulnerabilities by image, node, or platform. You can further view vulnerabilities by clusters, namespaces, deployments, node components, and image components. To access the dashboard, go to *Vulnerability Management* -> *Dashboard*. The dashboard is deprecated in {product-title-short} 4.5 and will be removed in a future release.
 
-* To view application vulnerabilities by namespace or deployment, or to view vulnerabilities in an image, in the {product-title-short} web portal, go to *Vulnerability Management (1.0)* -> *Dashboard*.
-* To view vulnerabilities in applications running on clusters in your system, go to *Vulnerability Management (2.0)* -> *Workload CVEs*. You can filter vulnerabilities by image, deployment, namespace, and cluster.
-
-include::modules/vulnerability-management-view-applications-vulnerability.adoc[leveloffset=+2]
-
-include::modules/vulnerability-management-view-image-vulnerability.adoc[leveloffset=+2]
-
-[role="additional-resources"]
-.Additional resources
-* xref:../../operating/search-filter.adoc#use-local-page-filtering_search-filter[Using local page filtering]
+The *Vulnerability Management* -> *Workload CVEs* page provides information about vulnerabilities in applications running on clusters in your system. You can view vulnerability information across images and deployments. The *Workload CVEs* page provides more advanced capabilities than the Dashboard for filtering, including the ability to view images and deployments with vulnerabilities and filter by image, deployment, namespace, cluster, CVE, component, and component source.
 
 include::modules/vulnerability-management20-view-cve.adoc[leveloffset=+2]
+include::modules/vulnerability-management-mark-false-positive.adoc[leveloffset=+2]
+include::modules/vulnerability-management-review-deferred.adoc[leveloffset=+2]
+include::modules/vulnerability-management-upgrade-component.adoc[leveloffset=+2]
+include::modules/vulnerability-management-export-workloads.adoc[leveloffset=+2]
 
-include::modules/vulnerability-management-view-infrastructure-vulnerability.adoc[leveloffset=+3]
+[id="using-the-vm-dashboard"]
+=== Using the Dashboard (deprecated) to view and manage vulnerabilities
```

Comment:
```suggestion
=== Using the Dashboard (deprecated) to view vulnerabilities
```
The Dashboard is read-only in 4.5, and all management functionality has been moved to the Workload, Node, and Platform CVE sections, as well as the Exception Management section.